docs/user/dashboard.asciidoc

```diff
@@ -34,54 +32,60 @@ dashboards are not visible. For more information, see <<xpack-security-authoriza
 [role="screenshot"]
 image::images/dashboard-read-only-badge.png[Example of Dashboard's read only access indicator in Kibana's header]
 
-[float]
-[[dashboard-getting-started]]
-=== Interact with dashboards
-
-When you open *Dashboard*, you're presented an overview of your dashboards. 
-If you don't have any dashboards, you can add 
-<<add-sample-data, sample data sets>>,
-which include pre-built dashboards. 
-
-Once you open a dashboard, you can filter the data
-by entering a search query, changing the time filter, or clicking 
-in the visualizations, searches, and maps. If a dashboard element has a stored query, 
-both queries are applied.
-
 --
 
 [[dashboard-create-new-dashboard]]
 == Create a dashboard
 
-To create a dashboard, you must have data indexed into {es}, an index pattern 
-to retrieve the data from {es}, and 
-visualizations, saved searches, or maps. If these don't exist, you're prompted to 
-add them as you create the dashboard.
+To create a dashboard, you must have data indexed into {es}, an index pattern
+to retrieve the data from {es}, and
+visualizations, saved searches, or maps. If these don't exist, you're prompted to
+add them as you create the dashboard, or you can add
+<<add-sample-data, sample data sets>>,
+which include pre-built dashboards.
 
 For an end-to-end example, see <<tutorial-build-dashboard, Building your own dashboard>>.
```

Comment:
Maybe we don't need this line?  Looks kinda odd with back-to-back one-sentence paragraphs.